app/src/main/java/org/fossasia/openevent/app/utils/SearchUtils.java

```diff
@@ -0,0 +1,28 @@
+package org.fossasia.openevent.app.utils;
+
+import org.fossasia.openevent.app.data.models.Attendee;
+
+import me.xdrop.fuzzywuzzy.FuzzySearch;
+
+public class SearchUtils {
+
+    /**
+     * Token sort partial ratio returns a value in percentage about similarity
+     * between two strings. The same is tested and found that
+     * greater than 60 does well. So we are checking similarity of a query with
+     * first name, last name and email of an attendee if at least one of them is greater than 60.
+     *
+     * @param attendee to be filtered
+     * @param query search query
+     * @return boolean saying filter or not
+     */
+    public static boolean shallFilterAttendee(Attendee attendee, CharSequence query) {
```

Comment:
Make this a generic method, not just for an attendee. First argument may be charsequence and second one String varargs